src/adk/command_list.py

```diff
@@ -254,12 +258,12 @@ def applications_delete(
     deleted_completely = processor.applications_delete(application_name=application_name,
                                                        application_path=application_path)
     if deleted_completely:
-        typer.echo("Application deleted successfully")
+        typer.echo("Application deleted successfully.")
```

Comment:
It was a choice to not print the . (dot)
We also do not print them in the exceptions, now is it inconsistent
Some (not all) cli's we tried don't end their responses with end of line dots (dos, bash, kubectl), however others (git) do.

Reply:
Good grammar is good grammar regardless of who else does it... but I take your point, it should be consistent.